packages/e2e-tests/run.ts

```diff
@@ -51,6 +55,11 @@ if (missingEnvVar) {
 
 const envVarsToInject = {
   REACT_APP_E2E_TEST_DSN: process.env.E2E_TEST_DSN,
+  NEXT_PUBLIC_E2E_TEST_DSN: process.env.E2E_TEST_DSN,
+  // Below are required for sentry-cli to work
+  SENTRY_ORG: process.env.E2E_TEST_SENTRY_ORG_SLUG,
+  SENTRY_PROJECT: process.env.E2E_TEST_SENTRY_TEST_PROJECT,
+  SENTRY_AUTH_TOKEN: process.env.E2E_TEST_AUTH_TOKEN,
```

Comment:
These shouldn't be needed in this PR yet. Let's remove them here until we actually get to the part where we're thesting the uploading of source maps.

Reply:
👍  [d9c2cf0](https://github.com/getsentry/sentry-javascript/pull/6669/commits/d9c2cf044557d49fe8175515fa2f8e32d87cc250)